middleware/diesel/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "gotham_middleware_diesel"
-version = "0.1.0"
-authors = ["Shaun Mangelsdorf <s.mangelsdorf@gmail.com>",
-           "Bradley Beddoes <bradleybeddoes@gmail.com>"]
+version = "0.4.0-dev"
```

Comment:
I think you mentioned this, but I don't know if we need to keep the versions in sync with Gotham for the sake of it... I'm leaning to `0.1.0` making more sense as a start point here - not sure if you agree?

Reply:
The issue that I encountered was that the version of gotham declared by the middleware must match the version of gotham declared by your project (or cargo has be able to resolve them to the same version at least). This leaves the user with the awkward question of "which version of the middleware must I use?" - e.g. middleware X version "0.1" is compatible with my gotham version of "0.3", and middleware Y version "0.5" is. When gotham gets a new release, a new version of middlewares needs to be released that bumps the dependency version (with our pre-1.0 versions anyway). It all seemed easier to keep track of if the version numbers were just in sync - at least the "major" version.

Reply:
I've set this version back to "0.1.0".